index.html

```diff
@@ -4789,10 +4789,10 @@ <h3>Verification</h3>
         <p>
 This section contains an algorithm that [=conforming verifier
 implementations=] MUST run when verifying a [=verifiable credential=] or a
-[=verifiable presentation=]. This algorithm takes inputs of a media
-type ([=string=] |inputMediaType|) paired with either a sequence of bytes
-([=byte sequence=] |inputBytes|) or a document ([=map=] |inputDocument|),
-and returns a [=map=] that contains the following:
+[=verifiable presentation=]. This algorithm takes a media
+type ([=string=] |inputMediaType|) and secured data
+([=byte sequence=] or [=map=] |securedData|) and returns a [=map=] that
```

Comment:
We might want this to be "inputData" instead -- since the algorithm is used to determine the secured status? I can live with either name or a third name that might be better than these.

```suggestion
([=byte sequence=] or [=map=] |inputData|) and returns a [=map=] that
```